public/pages/CreateMonitor/components/VisualGraph/VisualGraph.js

```diff
@@ -87,35 +89,37 @@ export default class VisualGraph extends Component {
     const yTitle = getYTitle(values);
     const leftPadding = getLeftPadding(yDomain);
     const markData = getMarkData(data);
-    const aggregationTitle = getAggregationTitle(values);
+    const title = aggregationType
+      ? `${aggregationType?.toUpperCase()} OF ${fieldName}`
+      : 'COUNT of documents';
```

Comment:
I think we will not reach here. Unless we push this `COUNT of documents` into `values.aggregations`

Reply:
I think this is because to use the updated title if no additional metric aggregation is defined the graph title will show up as this: 
![image](https://user-images.githubusercontent.com/71157062/130305658-32c1857b-b987-452e-9006-0d7c2a5b0e61.png)